scripts/filter_and_tag_services_using_opslevel_cli/opslevel_cli_commands.md

```diff
@@ -0,0 +1,65 @@
+# Filter and Tag Services Using the opslevel-cli
+
+The opslevel-cli commands and steps below can be used to filter services by a particular property and tag the resulting services. A subsequent filter can be created in OpsLevel based on the tag assigned.
+
+In this example, services are filtered based on their `createdAt` date if they were created after a specific date and then tagged with the `created-after` tag.
+
+Some steps using the opslevel-cli can be chained, but kept separate for simplicity.
+
+# Requirements
+
+* opslevel-cli installed https://github.com/OpsLevel/cli
+* jq https://stedolan.github.io/jq/
+
+# Steps and commands
+
```

Comment:
These steps can be simplified and step 2 can be dropped. Here is a simplified workflow.

```bash
# Date in format YYYY-MM-DD for year-month-day
DATETIME=$(date +"%Y-%m-%d")
opslevel list services -o json > services_$DATETIME.json

UNIX_TIMESTAMP=$(date +%s)
cat services_$DATETIME.json | jq -r '.[] | if .timestamps.createdAt | split("T").[0] + "T00:00:00Z" | fromdateiso8601 > env.UNIX_TIMESTAMP then .id else empty end' > service_ids_output.txt
```

Reply:
The UNIX_TIMESTAMP is to be determined by the end-user. The use case was to choose some date in the past, not `now`.

Reply:
I tried to make generating the UNIX_TIMESTAMP easier, but it introduces more complexity.